### ISSUE TITLE
keep adding plotting mission every 5 seconds

### ISSUE THREAD

**Issue by asukahan** [wolfrage76/windows-plotman#9]
by adding plotman to the daemon folder and running
`python plotman.py interactive`

window opened and seems fine, then plot mission added, log file and temp files appears, but it won't appear in the mission list. and missions kept added by every 5 seconds refresh.

**Comment by stubmirror** [comment]
same issue, dosnt seem to respect the rules set in config and just keeps adding plots every 5sec

**Comment by wolfrage76** [comment]
Did you change chia.exe to chia2.exe like the readme says, to not interfere
with existing plots while the code is still in testing, like the readme
says?

**Comment by stubmirror** [comment]
yes but i also noticed the running plots dont show up in the overview so iam guessing something else is wrong, iam using python 3.9 if that matters and installed all the prereqs with pip.
![powershell_qhQdQXfryi](https://user-images.githubusercontent.com/83403205/116542819-66c78780-a8ed-11eb-909b-034f64eae6bc.png)

**Comment by pvjulien** [comment]
Does anyone have a solution to this? I have a similar problem (and also to issue #8) and can't figure what's going wrong.

Thank you!
